src/components/Dropdown/Dropdown.tsx

```diff
@@ -34,6 +34,8 @@ export class Dropdown extends React.PureComponent<Props, State> {
   private getUniqueID = createUniqueIDFactory('Dropdown');
   private activatorContainer: HTMLElement | null;
   private id = this.getUniqueID();
+  private dropdownEle: any = null;
+  private dropdownOffset: any = { height: 0, width: 0 };
```

Comment:
@dheeraja00 Again, not a fan of `: any`.  I think you can remove `: any` and typescript might infer the type.  Even though this is private, linting and ts is not of much help if we use any.  It ignores type checking for that var, defeating the whole point.